tensorboard/plugins/mesh_visualizer/http_api.md

```diff
@@ -0,0 +1,65 @@
+# Mesh plugin HTTP API
+
+The mesh plugin name is `mesh`, so all its routes are under
+`/data/plugin/mesh`.
+
+## `/data/plugin/mesh/tags`
+
+Retrieves an index of tags containing mesh data.
+
+Returns a dictionary mapping from `runName` (quoted string) to
+dictionaries that map a `tagName` (quoted string) to an object
+containing that tag’s `displayName` and `description`, the latter of
+which is a string containing sanitized HTML to be rendered into the DOM.
+Here is an example:
```

Comment:
This does not render properly. Please use the preview functionality in
GitHub to inspect your changes:

![Screenshot of “preview” button in diff viewer](https://user-images.githubusercontent.com/4317806/57115869-45376380-6d06-11e9-9e5f-c7a6b365897f.png)

In this case, you should surround the code in triple-backticks:

``````markdown
[…] to be rendered into the DOM. Here is an example:

```json
    {
        "train_run": {
            "mesh_color_tensor": {
                "samples": 1
            }, 
            "point_cloud": {
                "samples": 1
            }
        }
    }
```

Note that runs […]
``````

Likewise below.